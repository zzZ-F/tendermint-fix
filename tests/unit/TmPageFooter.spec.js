import { mount } from "@vue/test-utils"
import { TmPageFooter } from "@/index"

describe("TmPageFooter", () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(TmPageFooter)
  })

  it("has the expected html structure", () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
