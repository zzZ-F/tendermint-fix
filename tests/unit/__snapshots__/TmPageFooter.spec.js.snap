// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmPageFooter has the expected html structure 1`] = `
<footer
  class="tm-page-footer"
>
  <div
    class="container"
  />
</footer>
`;
